templates/theme/reports/a11y-1.json

```diff
@@ -0,0 +1,635 @@
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:forEvaluation",
+      "acr:relevancy": "acr:forEvaluation",
+      "dct:description": "",
+      "earl:outcome": "earl:passed"
+    },
+    {
+      "earl:test": "WCAG21:non-text-contrast",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:forEvaluation",
+      "acr:relevancy": "acr:forEvaluation",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable"
+    },
+    {
+      "earl:test": "WCAG21:text-spacing",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:forEvaluation",
+      "acr:relevancy": "acr:forEvaluation",
+      "dct:description": "",
+      "earl:outcome": "earl:passed"
+    },
+    {
+      "earl:test": "WCAG21:content-on-hover-or-focus",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable"
+    },
+    {
+      "earl:test": "WCAG21:keyboard",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:forEvaluation",
+      "acr:relevancy": "acr:forEvaluation",
+      "dct:description": "",
+      "earl:outcome": "earl:passed"
+    },
+    {
+      "earl:test": "WCAG21:no-keyboard-trap",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:forEvaluation",
+      "acr:relevancy": "acr:forEvaluation",
+      "dct:description": "",
+      "earl:outcome": "earl:passed"
+    },
+    {
+      "earl:test": "WCAG21:character-key-shortcuts",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable"
+    },
+    {
+      "earl:test": "WCAG21:timing-adjustable",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable"
+    },
+    {
+      "earl:test": "WCAG21:pause-stop-hide",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable"
+    },
+    {
+      "earl:test": "WCAG21:three-flashes-or-below-threshold",
```

Comment:
This SC is always applicable when making an assessment. 
The test consist into loading the page and ensuring the flashes are below the threshold. There is always at least one which occur on page load.